mlflow/pytorch/__init__.py

```diff
@@ -128,6 +128,15 @@ def log_model(
     """
     Log a PyTorch model as an MLflow artifact for the current run.
 
+        .. warning::
+
+            Log the model with a signature to avoid inference errors.
+            For models without signatures, the MLflow Model Server relies on the default inferred
+            data type from NumPy. However, PyTorch often expects different defaults, particularly
+            when parsing floats. Include the signature to ensure that the model is logged with the
+            correct data type so that the MLflow model server can correctly produce valid input
```

Comment:
My bad here, since I made the earlier suggestion, but could we actually do "provide" instead of "produce"

Reply:
Ack, done.